### ISSUE TITLE
It doesn't get any matadata from Twitter.

### ISSUE THREAD

**Issue by lipis** [samholmes/node-open-graph#28]
At the moment for any Twitter URL the only thing that is being returned is:

```json
{
  "site_name": "Twitter",
  "title": ""
}

```

Probably something changed on the Twitter side.

**Comment by samholmes** [comment]
Taking this URL as an example: https://twitter.com/elonmusk/status/1224938416795717634

There appears to be only `<meta property="og:site_name" content="Twitter" />` from the source and no `<title>` tag from the source. Twitter's web app must be an SPA without SSR.

In order to get the right data, client-side rendering would need to be introduced into the data processing pipeline.

**Comment by ffflorian** [comment]
Seems like Twitter is not rendering `og` tags anymore, see [OpenGraphCheck results](https://opengraphcheck.com/result.php?url=https%3A%2F%2Ftwitter.com%2Felonmusk%2Fstatus%2F1224938416795717634). The site wants you to execute JavaScript, if you don't do that, you just get a landing page.

But it works if you set `Twitterbot/1.0` as User-Agent. :)

**Comment by hannasage** [comment]
Yep, the above comments are correct. Twitter is running as a SPA and does not provide `meta` tags as a part of the web application, so Twitter links won't return `og` metadata. 

See: https://stackoverflow.com/questions/62526483/twitter-website-doesnt-have-open-graph-tags
